types/apollo-upload-client/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "dependencies": {
-        "@apollo/client": "^3.1.3",
-        "graphql": "^15.3.0"
+        "@apollo/client": "^3.5.6",
+        "graphql": "^15.8.0"
```

Comment:
should these ones be same as original versions?
```ts
"@apollo/client": "^3.0.0",
"graphql": "14 - 16"
```
https://github.com/jaydenseric/apollo-upload-client/blob/master/package.json